Mathlib/RingTheory/Length.lean

```diff
@@ -90,6 +90,32 @@ lemma Module.length_ne_top [IsArtinian R M] [IsNoetherian R M] : Module.length R
   rw [Module.length_ne_top_iff, isFiniteLength_iff_isNoetherian_isArtinian]
   exact ⟨‹_›, ‹_›⟩
 
+lemma Module.length_submodule {N : Submodule R M} :
+    Module.length R N = Order.height N := by
+  apply WithBot.coe_injective
+  rw [Order.height_eq_krullDim_Iic, coe_length, Order.krullDim_eq_of_orderIso (Submodule.mapIic _)]
+
+lemma Module.length_quotient {N : Submodule R M} :
+    Module.length R (M ⧸ N) = Order.coheight N := by
+  apply WithBot.coe_injective
+  rw [Order.coheight_eq_krullDim_Ici, coe_length,
+    Order.krullDim_eq_of_orderIso (Submodule.comapMkQRelIso N)]
+  rfl
```

Comment:
Thank you! Can you please change [Submodule.comapMkQRelIso](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/Quotient/Basic.html#Submodule.comapMkQRelIso) to use `Set.Ici`? That's why I didn't find it, and it should also get rid of the `rfl`.